api/authenticators/onelogin_duo/onelogin.go

```diff
@@ -2,89 +2,34 @@ package oneloginduo
 
 import (
 	"context"
-	"strings"
 
 	"github.com/riotgames/key-conjurer/api/settings"
 
 	"github.com/rnikoopour/onelogin"
-	"github.com/sirupsen/logrus"
 )
 
 // OneLogin is a wrapper around the onelogin library
 type OneLogin struct {
 	ReadUserClient *onelogin.Client
 	SamlClient     *onelogin.Client
-	logger         *logrus.Entry
 }
 
-type OneLoginApp struct {
-	app onelogin.App
-}
-
-func (app OneLoginApp) ID() int64 {
-	return app.app.ID
-}
-
-func (app OneLoginApp) Name() string {
-	return app.app.Name
-}
-
-// NewOneLogin creates a new onelogin client using the providing settings
-//  and logs with provided logger
-func NewOneLogin(settings *settings.Settings, logger *logrus.Entry) *OneLogin {
+// New creates a new onelogin client using the providing settings
+func newOneLogin(settings *settings.Settings) *OneLogin {
 	var readUserClient = onelogin.New(settings.OneLoginReadUserID, settings.OneLoginReadUserSecret,
 		settings.OneLoginShard, settings.OneLoginSubdomain)
 	var samlClient = onelogin.New(settings.OneLoginSamlID, settings.OneLoginSamlSecret,
 		settings.OneLoginShard, settings.OneLoginSubdomain)
 	return &OneLogin{
 		ReadUserClient: readUserClient,
 		SamlClient:     samlClient,
-		logger:         logger}
-}
-
-// AuthenticateUser validates the user against OneLogin
-func (o *OneLogin) AuthenticateUser(username, password string) (*onelogin.AuthenticatedUser, error) {
```

Comment:
This was inlined due to being so simple, with the error message handled by the caller.